testutil/ciao-down/ciao_down.go

```diff
@@ -137,41 +137,46 @@ func checkDirectory(dir string) error {
 	return nil
 }
 
-func prepareFlags(ws *workspace) (*instance, bool, string, string, error) {
+func prepareFlags(ws *workspace) (*instance, *workload, bool, error) {
 	var debug bool
-	var runCmd, vmType string
+	var vmType string
 	var m mounts
 	var p ports
 
 	memGiB, CPUs := getMemAndCpus()
 	fs := flag.NewFlagSet("prepare", flag.ExitOnError)
 	vmFlags(fs, &memGiB, &CPUs, &m, &p)
 	fs.BoolVar(&debug, "debug", false, "Enables debug mode")
-	fs.StringVar(&runCmd, "runcmd", "", "Path to a file containing additional commands to execute when preparing the VM")
-	fs.StringVar(&vmType, "vmtype", CIAO, "Type of VM to launch. "+CIAO+" or "+CLEARCONTAINERS)
+	fs.StringVar(&vmType, "vmtype", CIAO, "Type of VM to launch.")
 
 	if err := fs.Parse(flag.Args()[1:]); err != nil {
-		return nil, false, "", "", err
+		return nil, nil, false, err
```

Comment:
I guess cause it's not used anywhere else in ciao-down right now.  Migrating to a errors wrap would need to be a separate PR.